apps/ensindexer/test/ids.test.ts

```diff
@@ -42,6 +62,14 @@ describe("ids", () => {
       });
     });
 
+    describe("parseResolverId", () => {
+      it("should parse subgraph-compatible format", () => {
+        const node = namehash("vitalik.eth");
+        const resolverId = makeResolverId(CHAIN_ID, zeroAddress, node);
+        expect(parseResolverId(resolverId)).toEqual({ chainId: null, address: zeroAddress, node });
+      });
```

Comment:
Let's move this test case in to the `describe("parseResolverId", () => {` code block from line 30.